src/SpriteAnimation/SpriteAnimation.js

```diff
@@ -493,8 +502,14 @@ module.exports = kind(
 	/**
 	* @private
 	*/
-	_generateKeyframe: function (percent, x, y) {
-		return (Math.ceil(percent*1000000) / 10000) +'%	{ -webkit-transform: translate3d('+ x +'px, '+ y +'px, 0);	transform: translate3d('+ x +'px, '+ y +'px, 0); }\n';
+	_generateKeyframe: function (percent, x, y, prefix) {
```

Comment:
That was actually how the first version of this PR looked, but it's unnecessary and wasteful to have -webkit- rules inside a @-moz-keyframes block, and vice versa. We also need standards-only blocks generatable for minimum rulesets. We Also need -webkit- prefixed block without -webkit- prefixed rules for the dozen versions of chrome that DO support standard transform but DON'T support @keyframe rule.

This creates the most efficient and minimally required source code.